devices/Ssd13xx/Ssd13xx.nuspec

```diff
@@ -22,6 +22,7 @@
       <dependency id="nanoFramework.CoreLibrary" version="1.14.2" />
       <dependency id="nanoFramework.System.Device.I2c" version="1.1.9" />
       <dependency id="nanoFramework.System.Device.Gpio" version="1.1.28" />
+	  <dependency id="nanoFramework.System.Math" version="1.5.29" />
```

Comment:
```suggestion
    <dependency id="nanoFramework.System.Math" version="1.5.29" />
```

Reply:
Good grief.  Forgot to check that.  It's been fixed.